### DISCUSSION TITLE
rabitmq做了镜像队列集群 配置和代码需要什么改动？

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#243] [02.提问]
rabitmq做了镜像队列集群 配置和代码需要什么改动？

**Comment by zuihou** [comment]
lamp-cloud 只对 rabbitMq 封装了一个 enabled 参数，控制如何启用或禁用mq。


至于集群如何配置，请参看 spring boot 配置 rabbitmq 集群。
